packages/processor/test/specimens/composes/external-compose-via-local-compose3.css

```diff
@@ -0,0 +1,11 @@
+.ruleWithSingleCompose {
+    composes: red from "../deps/classes.css";
+}
+
+.ruleWithLocalComposes {
+    composes: ruleWithSingleCompose;
+}
+
+.ruleWithLocalComposes {
+    composes: blue from "../deps/classes.css";
```

Comment:
Duplication of `.ruleWithLocalComposes` intended?

Reply:
https://github.com/tivac/modular-css/pull/892#discussion_r991872610